### PR TITLE
Use cksum instead of sha1sum for checksums

### PR DESCRIPTION
**Pull request by blueyed**
Fixes https://github.com/Tarrasch/zsh-autoenv/issues/53.

We might to migrate existing checksums?!
Not sure if it's worth the effort though..

### CODE REVIEW

File: autoenv.zsh [hunk 1]
```diff
@@ -173,15 +173,15 @@ _autoenv_debug() {
...
-  local env_shasum=${2:-}
-  if [[ -z $env_shasum ]]; then
+  local env_cksum=${2:-}
+  if [[ -z $env_cksum ]]; then
     if ! [[ -e $env_file ]]; then
       echo "Missing file argument for _autoenv_hash_pair!" >&2
       return 1
     fi
-    env_shasum=$(sha1sum $env_file | cut -d' ' -f1)
+    env_cksum=${(j:.:)${:-$(cksum "$env_file")}[1,2]}
```
Comment (Tarrasch) [inline autoenv.zsh]: Adding an explicit comment explaining what `(j:.:)` does would be nice. zsh is so not readable...

**Comment by blueyed** [comment]
> Can we just double-check with som Mac-user that they actually have `cksum` available?

Should be there according to @rspeed.
@jasisk
Can you confirm this?

File: autoenv.zsh (hunk 1 above)
Comment (blueyed) [inline autoenv.zsh]: Fair point, added.
Should be squash-merged.

**Comment by jasisk** [comment]
> Can you confirm this?

Confirmed. 👍

**Comment by blueyed** [comment]
Thanks!

**Comment by rspeed** [comment]
Well crap. I made my own implementation earlier today and got sidetracked before I could push it. Gonna open a PR anyway since there's one difference that I think is probably important.

**Pull request merged**